zerver/tests/test_soft_deactivation.py

```diff
@@ -632,6 +633,7 @@ def assert_num_possible_users(
                         realm_id=realm_id,
                         stream_id=stream_id,
                         possible_wildcard_mention=possible_wildcard_mention,
+                        possibly_topic_wildcard_mentioned_user_ids=possibly_topic_wildcard_mentioned_user_ids,
                         possibly_mentioned_user_ids=possibly_mentioned_user_ids,
                     )
                 ),
```

Comment:
I would be adding the following code block at line 771 of this file in later commits:

```python
 # Test UserMessage row is created while user is deactivated if there is a topic wildcard mention
 # i.e. @topic
    assert_stream_message_sent_to_idle_user(
         "Test @**topic** mention", possibly_topic_wildcard_mentioned_user_ids={long_term_idle_user.id}
    )
    assert_stream_message_not_sent_to_idle_user("Test @**bogus** mention")

```
Not adding here as `assert_stream_message_sent_to_idle_user` (line 643) tests things other than `len(get_subscriptions_for_send_message())` which will be added in later commits, so tests can be added at that time.

Reply:
This has been addressed in `mention: Send notifications for @topic wildcard mentions.` commit.